Documentation/ApiOverview/SymfonyExpressionLanguage/Index.rst

```diff
@@ -180,7 +182,7 @@ All further arguments need to be provided by TypoScript. The above example could
        page.10.value = Matched
    [GLOBAL]
 
-In case a simple string like a title is returned this can be further checked:
+In case a simple string like a title is returned, this can be further checked:
```

Comment:
```suggestion
If a simple string like a title is returned, this can be further checked:
```

Reply:
If a simple string like a page title is returned, this can be further compared: